apps/dashboard/components/dashboard/copy-button.tsx

```diff
@@ -31,12 +35,12 @@ export function CopyButton({ value, className, src, ...props }: CopyButtonProps)
         copyToClipboardWithMeta(value, {
           component: src,
         });
-        setHasCopied(true);
+        setCopied(true);
```

Comment:
This can be fixed along with the introduction of the `useCopyToClipboard` hook if approved.

By the way, `copyToClipboardWithMeta` seems to not use `_meta` property. Is it intended to help debugging the component code? This is important to clarify if we decide to introduce a new hook.

Reply:
tbh I don't even know what the meta is referring to here, I think we can safely remove that